pkg/stanza/adapter/receiver_test.go

```diff
@@ -259,150 +255,6 @@ func benchmarkReceiver(b *testing.B, logsPerIteration int, batchingInput, batchi
 	require.NoError(b, rcv.Shutdown(b.Context()))
 }
 
-func BenchmarkReadLine(b *testing.B) {
```

Comment:
👍  I agree with removing this benchmark, we have other benchmarks on various levels (File consumer, File input, File Log receiver) and without this quirky setup with `b.N`  tied to input size.